UPGRADE-7.0.md

```diff
@@ -202,6 +202,14 @@ FrameworkBundle
  * Remove the integration of Doctrine annotations, use native attributes instead
  * Remove `EnableLoggerDebugModePass`, use argument `$debug` of HttpKernel's `Logger` instead
  * Remove `AddDebugLogProcessorPass::configureLogger()`, use HttpKernel's `DebugLoggerConfigurator` instead
+ * Set `framework.handle_all_throwables` config option default value to `true`
```

Comment:
This should be upgrade instructions. Let's recommend unsetting this values from the project when the value is the new default.

Reply:
The current message is roughly how we documented this previously
To be more consistent we could use
`Make the foo config option default to bar`

Reply:
Updated with Nicolas' format :+1: